build/test-threadSafety.ps1

```diff
@@ -0,0 +1,35 @@
+param(
+  [Parameter(Mandatory=$true)][string]$coyoteVersion="",
+  [Parameter(Mandatory=$true)][string]$targetFramework,
+  [Parameter(Mandatory=$true)][string]$methodNames
+)
+
+$rootDirectory = Split-Path $PSScriptRoot -Parent
+
+Write-Host "Install Coyote CLI."
+dotnet tool install --global Microsoft.Coyote.CLI
+
+Write-Host "Build OpenTelemetry.Tests project."
+dotnet build $rootDirectory\test\OpenTelemetry.Tests\OpenTelemetry.Tests.csproj

```

Comment:
I see in the workflow we do a restore + build and then in the powershell here we call build which will also build + restore. Seems like some duplicate effort. Also below where we call dotnet test that will also build + restore 😄